packages/pds/src/db/migrations/20230908T224408678Z-email-tokens.ts

```diff
@@ -0,0 +1,27 @@
+import { Kysely } from 'kysely'
+import { Dialect } from '..'
+
+export async function up(db: Kysely<unknown>, dialect: Dialect): Promise<void> {
+  const timestamp = dialect === 'sqlite' ? 'datetime' : 'timestamptz'
+  await db.schema
+    .createTable('email_token')
+    .addColumn('purpose', 'varchar', (col) => col.notNull())
+    .addColumn('did', 'varchar', (col) => col.notNull())
+    .addColumn('token', 'varchar', (col) => col.notNull())
+    .addColumn('requestedAt', timestamp, (col) => col.notNull())
```

Comment:
We haven't really used this in the pds yet, but nice!  May be worth a quick test if you haven't already that inserting a date then selecting it out results in two equal `Date`s, particularly if local server time is not GMT.  Essentially that kysely or the pg driver doesn't muck with the times in some unanticipated way.

Reply:
good point 👍 i'll run some tests

I went back and forth on if this should be a timestamp or string. I'm not opposed to just sticking with stirngs in the PDS (since we support both sqlite & postgres and it smooths over the difference). Thinking was "strings for records and timestamps for other stuff". But I suppose this leaves it in a pretty inconsistent state

Reply:
hmmm i think i'm talking myself into making this a string 😅